PRESUBMIT.py

```diff
@@ -290,7 +290,14 @@ def CheckLicense(input_api, output_api):
         ('Use std::string_view instead', ),
         True,
         [_THIRD_PARTY_EXCEPT_BLINK],  # Don't warn in third_party folders.
-    ), )
+    ),
+    BanRule(
+        'base::PathService::Get',
+        ('Prefer using base::PathService::CheckedGet() instead', ),
+        treat_as_error=False,
```

Comment:
I think we can enforce it, I don't see downsides.

Reply:
The reason I didn't make it an error is because I guess there might be cases where you want to detect and fail gracefully if you can't get a path, and it's not a catastrophic error.

Reply:
Ideally we would enforce it as error in tests.